src/views/Analysis.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { BigBoard } from '../components/Board/BigBoard/BigBoard';
+import { RouteComponentProps } from 'react-router';
+import { connect } from 'react-redux';
+import {
+  AppState,
+  Move,
+  Player,
+  SmallBoardInformation,
+} from '../state/AppState';
+import {
+  loadFinishedGame,
+  moveBackwardInHistory,
+  moveForwardInHistory,
+} from '../state/analysisGame/analysisGameActions';
+import './analysis.css';
+import { Point } from '../util/Point';
+import { HistoryButtons } from '../components/Analysis/HistoryButtons';
+import { MoveList } from '../components/Analysis/MoveList';
+
+interface AnalysisProps {
+  onLoad: (id: string) => void;
+  moveForwardInHistory: (numberOfMoves: number) => void;
+  moveBackwardInHistory: (numberOfMoves: number) => void;
+  reversedMoves?: Move[];
+  board: SmallBoardInformation[];
+  activeBoards: Point[];
+  currentPlayer: Player;
+  currentMove: number;
+}
+
+export class Analysis extends React.Component<
```

Comment:
I don't like this formatting from prettier. 
What do you think @RohrerF ? How could we change it?

Reply:
Do you like it now?

Reply:
Yes